### ISSUE TITLE
Test failures: GCStress time out in tracing pauseonstart, reverse tests

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/runtime#38524]
tracing\eventpipe\pauseonstart\pauseonstart\pauseonstart.cmd and tracing\eventpipe\reverse\reverse\reverse.cmd in various GCStress modes, such as
```
CoreCLR Windows_NT x64 Checked gcstress0xc_zapdisable @ Windows.10.Amd64.Open
CoreCLR Windows_NT x86 Checked gcstress0xc_zapdisable_heapverify1 @ Windows.10.Amd64.Open
CoreCLR Windows_NT x64 Checked gcstress0xc_zapdisable_heapverify1 @ Windows.10.Amd64.Open
```
(there may be more)

with time outs, e.g.:

```
        126.7s: Sending 'exit' to subprocess stdin
        131.8s: Subprocess didn't exit in 5 seconds!
        132.2s: System.TimeoutException: Subprocess didn't exit in 5 seconds
           at Tracing.Tests.Common.Utils.RunSubprocess(Assembly currentAssembly, Dictionary`2 environment, Func`1 beforeExecution, Func`2 duringExecution, Func`1 afterExecution) in /Users/runner/runners/2.171.1/work/1/s/src/coreclr/tests/src/tracing/eventpipe/common/IpcUtils.cs:line 120
        132.3s: Calling process.Kill()
        132.3s: Subprocess stdout: 
          0.6s: The runtime has been configured to pause during startup and is awaiting a Diagnostics IPC ResumeStartup command from a server at 'DOTNET_TRACE_TESTS_t4qv4kkd.j3d'.
          9.7s: Subprocess started!  Waiting for input...
        132.3s: Subprocess stderr: 
        132.6s: Test passed: False
```

These stress modes significantly slow down the process, so it is not surprising that a short 5 seconds timeout is not sufficient.

The timeouts should be _significantly_ increased. Or perhaps these should not be run under GCStress.

**Comment by BruceForstall** [comment]
PR to disable for GCStress: https://github.com/dotnet/runtime/pull/38525

**Comment by tommcdon** [comment]
@josalem

**Comment by josalem** [comment]
Thanks @BruceForstall!  PR should be out later today!

**Comment by josalem** [comment]
Actually this should be fixed by #38474 which removes the internal timeouts in favor of letting Helix timeout if something goes wrong.

**Comment by tommcdon** [comment]
closing per @josalem's comment that #38474 addresses this.  GCStress lengthens the time the test runs and so the test timeout was removed.
